> Automatic Validation ended with:
> 
> ```
>  2025-07-08T18:44:06.9297436Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [17.14.6 (June 2025), 17.14.6 (June 2025)]]
>  2025-07-08T18:44:06.9364261Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [17.14.6 (June 2025), 17.14.6 (June 2025)]]
> ```
> 
> (Automated response - build 1080.)

I will look into this. Additionally, I will be additional poc for Microsoft Visual Studio releases on WinGet.

--- a/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
+++ b/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
@@ -24,8 +24,6 @@ ExpectedReturnCodes:
   ReturnResponse: noNetwork
 Commands:
 - devenv
-AppsAndFeaturesEntries:
-- DisplayVersion: 17.14.8
 ElevationRequirement: elevatesSelf
 Installers:
 - Architecture: x64

--- a/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
+++ b/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
@@ -25,7 +25,7 @@ ExpectedReturnCodes:
 Commands:
 - devenv
 AppsAndFeaturesEntries:
-- DisplayVersion: 17.14.6 (June 2025)
+- DisplayVersion: 17.14.8 (July 2025)
 ElevationRequirement: elevatesSelf
 Installers:
 - Architecture: x64

--- a/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
+++ b/manifests/m/Microsoft/VisualStudio/2022/Community/17.14.8/Microsoft.VisualStudio.2022.Community.installer.yaml
@@ -25,7 +25,7 @@ ExpectedReturnCodes:
 Commands:
 - devenv
 AppsAndFeaturesEntries:
-- DisplayVersion: 17.14.8 (July 2025)
+- DisplayVersion: 17.14.8
 ElevationRequirement: elevatesSelf
 Installers:
 - Architecture: x64

